shared/actions/config/index.js

```diff
@@ -143,6 +142,7 @@ const bootstrap = (opts: $PropertyType<ConfigGen.BootstrapPayload, 'payload'>):
         if (!didInitialNav) {
           dispatch(async () => {
             if (getState().config.loggedIn) {
+              dispatch(LoginGen.createNavBasedOnLoginAndInitialState())
```

Comment:
i do want to clean up how this works but short term we need to call this more often sometimes